addons/account/i18n/account.pot

```diff
@@ -8823,6 +8823,13 @@ msgid ""
 "Please set at least one of the match texts to create a partner mapping."
 msgstr ""
 
+#. module: account
+#: code:addons/account/models/account_reconcile_model.py:0
+#, python-format
+msgid ""
+"The following regular expression is invalid to create a partner mapping: %s"
```

Comment:
It doesn't seem ordered alphabetically, how did you add the entry?

Reply:
I thought it had to be manually added to the "account.pot" file. I corrected that by following the right steps